.github/workflows/linux.yml

```diff
@@ -53,20 +53,20 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-          args: --all --bins --examples --tests
+          args: --workspace --bins --examples --tests
 
       - name: tests
         uses: actions-rs/cargo@v1
         timeout-minutes: 40
         with:
           command: test
-          args: --all --all-features --no-fail-fast -- --nocapture
+          args: --workspace --exclude=actix-tls --no-fail-fast -- --nocapture
```

Comment:
why exclude tls?

Reply:
Currently, we don't have any tests including doctests on actix-tls so I think it's okay to exclude for now to speed up.

Reply:
oh right 😅